ui-design-lib/build.gradle.kts

```diff
@@ -43,9 +43,6 @@ dependencies {
     implementation(libs.kotlinx.coroutines.core)
     implementation(projects.spackleLib)
 
-    // Material2 is only used for the EditText
-    implementation(libs.androidx.compose.material)
```

Comment:
Great to have support for the TextFields in Material 3! Hope we don't need other common buttons soon in our screens, as it still does not support a lot of UI components like Checkboxes, Switches, etc.